simulariumio/data_objects/custom_data.py

```diff
@@ -34,19 +36,25 @@ def __init__(
 
         Parameters
         ----------
-        spatial_unit_factor_meters : float
-            A float multiplier needed to convert spatial data
-            (including positions, radii, and box size) to meters
         box_size : np.ndarray (shape = [3])
             A numpy ndarray containing the XYZ dimensions
             of the simulation bounding volume
         agent_data : AgentData
             An object containing data for each agent
             at each timestep
+        time_unit_factor_seconds : float (optional)
+            A float multiplier needed to convert temporal data
+            (e.g. timeStepSize) to seconds
+            Default: 1.0
+        spatial_unit_factor_meters : float (optional)
+            A float multiplier needed to convert spatial data
+            (including positions, radii, and box size) to meters
+            Default: 1.0
```

Comment:
Would it be worth giving the example of "1e-9 = nanoseconds" here again for clarity?